website/snippets/_bigquery-permissions.md

```diff
@@ -0,0 +1,13 @@
+BigQuery's permission model differs from conventional databases like Snowflake and Redshift. dbt uer accounts need the following permissions to read from and create tables and <Term id="view">views</Term> in a BigQuery project:
+- BigQuery Data Editor
+- BigQuery User
+
+For BigQuery with <Constant name="fusion_engine" />, users also need:
+- BigQuery Read Session User (for Storage Read API access)
+
+For BigQuery DataFrames, users need these additional roles:
```

Comment:
We flip flop between roles and permissions in this section - should they both be the same language?
```suggestion
For BigQuery DataFrames, users need these additional roles:
```